flink-test-utils/src/main/java/org/apache/flink/test/util/TestEnvironment.java

```diff
@@ -80,7 +80,7 @@ public String getExecutionPlan() throws Exception {
 	private OptimizedPlan compileProgram(String jobName) {
 		Plan p = createProgramPlan(jobName);
 
-		Optimizer pc = new Optimizer(new DataStatistics());
+		Optimizer pc = new Optimizer(new DataStatistics(), this.executor.getConfiguration());
```

Comment:
For this and other tests, if we made static method to create new Configuration, we could make it clear this is just for local executor:

```
LocalExecutor.createConfigForLocalExecutor(this.executor);
```